engine/class_modules/apl/retribution_apl.inc

```diff
@@ -12,22 +12,24 @@ actions.precombat+=/variable,name=trinket_2_sync,op=setif,value=1,value_else=0.5
 actions.precombat+=/variable,name=trinket_priority,op=setif,value=2,value_else=1,condition=!trinket.1.has_use_buff&trinket.2.has_use_buff|trinket.2.has_use_buff&((trinket.2.cooldown.duration%trinket.2.proc.any_dps.duration)*(1.5+trinket.2.has_buff.strength)*(variable.trinket_2_sync))>((trinket.1.cooldown.duration%trinket.1.proc.any_dps.duration)*(1.5+trinket.1.has_buff.strength)*(variable.trinket_1_sync))
```

Comment:
`trinket.x.has_use_buff` seems to have broken in DF. Will need to be updated to use the trinket_x_buffs variable, otherwise will always prioritize slot 1.

Reply:
Thank you, this should now be fixed.